packages/react-cosmos-webpack/src/export.js

```diff
@@ -0,0 +1,58 @@
+import path from 'path';
+import webpack from 'webpack';
+import { argv } from 'yargs';
+import fs from 'fs-extra';
+import getWebpackConfig from './webpack-config';
+import getDefaultWebpackConfig from './default-webpack-config';
+import importModule from 'react-cosmos-utils/lib/import-module';
+import getCosmosConfig from 'react-cosmos-config';
+
+process.env.NODE_ENV = 'production';
+
+const moduleExists = modulePath => {
+  try {
+    return modulePath && require.resolve(modulePath) && true;
+  } catch (err) {
+    return false;
+  }
+};
+
+module.exports = function startExport() {
+  const cosmosConfigPath = argv.config;
+  const cosmosConfig = getCosmosConfig(cosmosConfigPath);
+
+  const {
+    webpackConfigPath
+  } = cosmosConfig;
+
+  let userWebpackConfig;
+  if (moduleExists(webpackConfigPath)) {
+    console.log(`[Cosmos Export] Using webpack config found at ${webpackConfigPath}`);
+    userWebpackConfig = importModule(require(webpackConfigPath));
+  } else {
+    console.log('[Cosmos Export] No webpack config found, using default configuration');
+    userWebpackConfig = getDefaultWebpackConfig();
+  }
+
+  const cosmosWebpackConfig = getWebpackConfig(userWebpackConfig, cosmosConfigPath, true);
+  const compiler = webpack(cosmosWebpackConfig);
+
+  compiler.run((err, stats) => {
+    if (err) {
+      console.error('Export Failed! See error below:');
+      console.error(err);
+    } else {
+      console.log(stats);
+      try {
+        fs.copySync(path.join(__dirname, 'static/favicon.ico'), `${cosmosWebpackConfig.output.path}/favicon.ico`);
```

Comment:
> Index of the user, used for the contents of the loader frame (this one is generated by the export, but should be placed in the loader/ dir, next to the bundle.js file)

Seems to work fine without this one, but then I use a web server that is redirecting to the index.html of the playground. Can you elaborate on when the `index of the user` comes into play?